YIF.Core.Data/Entities/ExamRequirement.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Text;
+
+namespace YIF.Core.Data.Entities
+{
+    [Table("ExamRequirements")]
+    public class ExamRequirement: BaseEntity
+    {
+        public string ExamId { get; set; }
+
+        public string SpecialtyInUniversityDescriptionId { get; set; }
+        
+        public double MinimumScore { get; set; }
+        
+        public Exam Exam { get; set; }
+        
+        public SpecialtyInUniversityDescription SpecialtyInUniversityDescription { get; set; }
```

Comment:
We need the only score. Links on exam or SpecialtyInUniversityDescription won`t be used.